40_replace_egrep_fgrep_sigs.dpatch by David Weinehall <tao@debian.org>

Replace egrep by grep -E, fgrep by grep -F, and numerical signals
by signal names

---
 makeinfo/tests/cond |   16 ++++++++--------
 util/fixfonts       |    2 +-
 util/gen-dir-node   |    6 +++---
 util/texi2dvi       |    2 +-
 4 files changed, 13 insertions(+), 13 deletions(-)

Index: texinfo-4.11/makeinfo/tests/cond
===================================================================
--- texinfo-4.11.orig/makeinfo/tests/cond	2005-05-15 01:46:32.000000000 +0200
+++ texinfo-4.11/makeinfo/tests/cond	2007-10-30 08:31:25.000000000 +0100
@@ -6,28 +6,28 @@
 
 # Default Info output.
 ../makeinfo -o cond.out $srcdir/cond.txi || exit 1
-egrep 'This is (ifnothtml|ifinfo|ifnottex) text' cond.out >/dev/null \
+grep -E 'This is (ifnothtml|ifinfo|ifnottex) text' cond.out >/dev/null \
   || exit 2
-test `fgrep -c ' text.' cond.out` -eq 3 || exit 3
+test `grep -F -c ' text.' cond.out` -eq 3 || exit 3
 
 # Default HTML output.
 ../makeinfo --no-split --html -o cond.out $srcdir/cond.txi || exit 1
-egrep 'This is (html|ifhtml|ifnotinfo|ifnottex) text' cond.out >/dev/null \
+grep -E 'This is (html|ifhtml|ifnotinfo|ifnottex) text' cond.out >/dev/null \
  || exit 2
-test `fgrep -c ' text.' cond.out` -eq 4 || exit 3
+test `grep -F -c ' text.' cond.out` -eq 4 || exit 3
 
 # --ifhtml off, --ifinfo off, --iftex off.
 ../makeinfo --no-ifhtml --no-ifinfo --no-iftex -o cond.out $srcdir/cond.txi || exit 1
-egrep 'This is ifnot(html|info|tex) text' cond.out >/dev/null \
+grep -E 'This is ifnot(html|info|tex) text' cond.out >/dev/null \
  || exit 2
-test `fgrep -c ' text.' cond.out` -eq 3 || exit 3
+test `grep -F -c ' text.' cond.out` -eq 3 || exit 3
 
 # Do we really need to test all the other permutations?
 
 # --ifhtml on, --ifinfo on, --iftex on.
 ../makeinfo --ifhtml --ifinfo --iftex -o cond.out $srcdir/cond.txi || exit 1
-egrep 'This is (html|ifhtml|ifinfo|tex|iftex) text' cond.out >/dev/null \
+grep -E 'This is (html|ifhtml|ifinfo|tex|iftex) text' cond.out >/dev/null \
  || exit 2
-test `fgrep -c ' text.' cond.out` -eq 5 || exit 3
+test `grep -F -c ' text.' cond.out` -eq 5 || exit 3
 
 rm -f cond.out cond.info
Index: texinfo-4.11/util/fixfonts
===================================================================
--- texinfo-4.11.orig/util/fixfonts	2005-05-15 01:46:35.000000000 +0200
+++ texinfo-4.11/util/fixfonts	2007-10-30 08:30:30.000000000 +0100
@@ -33,7 +33,7 @@
 tempfile2="${TMPDIR}/circ2$$"
 
 # EXIT SIGHUP SIGINT SIGQUIT SIGTERM
-#trap 'rm -f "${tempfile}" "${tempfile2}"' 0 1 2 3 15
+#trap 'rm -f "${tempfile}" "${tempfile2}"' 0 HUP INT QUIT TERM
 
 # Find all the fonts with names that include `circle'.
 (cd "${texfontdir}"; find . -name '*circle*' -print > "${tempfile}")
Index: texinfo-4.11/util/gen-dir-node
===================================================================
--- texinfo-4.11.orig/util/gen-dir-node	2005-05-15 01:46:35.000000000 +0200
+++ texinfo-4.11/util/gen-dir-node	2007-10-30 08:30:30.000000000 +0100
@@ -79,7 +79,7 @@
 ### then generate entries for those in the same way, putting the info for 
 ### those at the end....
 
-infofiles=`(cd ${INFODIR}; /bin/ls | grep -v '\-[0-9]*$' | egrep -v '^dir$|^dir\.info$|^dir\.orig$')`
+infofiles=`(cd ${INFODIR}; /bin/ls | grep -v '\-[0-9]*$' | grep -E -v '^dir$|^dir\.info$|^dir\.orig$')`
 
 # echoing gets clobbered by backquotes; we do it the hard way...
 lines=`wc $SKELETON | awk '{print $1}'`
@@ -166,8 +166,8 @@
 
 # Sort remaining files by INFO-DIR-SECTION.
 prevsect=
-filesectdata=`(cd ${INFODIR}; fgrep INFO-DIR-SECTION /dev/null ${infofiles} | \
-	      fgrep -v 'INFO-DIR-SECTION Miscellaneous' | \
+filesectdata=`(cd ${INFODIR}; grep -F INFO-DIR-SECTION /dev/null ${infofiles} | \
+	      grep -F -v 'INFO-DIR-SECTION Miscellaneous' | \
 	      sort -t: -k2 -k1 | tr ' ' '_')`
 for sectdata in ${filesectdata}; do
   file=`echo ${sectdata} | cut -d: -f1`
Index: texinfo-4.11/util/texi2dvi
===================================================================
--- texinfo-4.11.orig/util/texi2dvi	2007-10-30 08:23:49.000000000 +0100
+++ texinfo-4.11/util/texi2dvi	2007-10-30 08:32:34.000000000 +0100
@@ -1623,7 +1623,7 @@
 	     sed 's,/,!,g'`
   esac
   # Remove it at exit if clean mode.
-  trap "cleanup" 0 1 2 15
+  trap "cleanup" 0 HUP INT TERM
 
   ensure_dir "$build_dir" "$t2ddir"
 
